broker-daemon/models/order.js

```diff
@@ -470,6 +470,20 @@ class Order {
       lte: `${blockOrderId}${DELIMITER}${UPPER_BOUND}`
     }
   }
+
+  /**
+   * Create a set of options that can be passed to a Level `createReadStream` call
+   * that limits the set to orders that belong to the given range of blockOrderIds.
+   * @param {string} firstId - blockOrderId of the start of our block order range
+   * @param {string} lastId - blockOrderId of the end of our block order range
+   * @returns {Object} Options object that can be used in {@link https://github.com/Level/levelup#createReadStream}
+   */
+  static rangeForIds (firstId, lastId) {
```

Comment:
don't we have this defined somewhere else?

Reply:
we have `rangeForBlockOrder` above this code but it doesn't allow for a range. If we want to consolidate these methods, we could

Reply:
just saw your comment here: https://github.com/sparkswap/broker/pull/527/files#r288694181 this is what I will do

Reply:
https://github.com/sparkswap/broker/pull/529